modules/itemManager.ts

```diff
@@ -212,6 +210,70 @@ class ItemManager {
     await db.items.update(id, { isPurchased: !!target.checked });
     await this.populateItems();
   }
+
+  /** Toggle all items in a list as 'in cart'/purchased/selected */
+  async toggleAllListItems(event: Event) {
+    const target = event.target as HTMLInputElement;
+    const shouldMarkAll = target.checked;
+
+    const listId = this.currentListId;
```

Comment:
If we're storing the current list id in the ItemManager class.... what do we really need the ListManager class for? I think ItemManager should probably reference the list id that lives in the ListManager.